src/web/handlers.rs

```diff
@@ -25,6 +25,7 @@ pub async fn get_tiles(
     trace!("get_tiles");
 
     let cinfo = request.connection_info();
+    let settings = state.settings.clone();
```

Comment:
nit: we don't need a clone here, see [my branch](https://github.com/mozilla-services/contile/compare/feat/101-loc-head...feat/101-loc-header-ref)

Reply:
also fixing the clippy that didn't trigger on my system.